workflow/source.go

```diff
@@ -138,6 +148,37 @@ func (d Document) IsRemote() bool {
 	return getFileStatus(d.Location) == fileStatusRemote
 }
 
+func (d Document) IsSpeakeasyRegistry() bool {
+	return strings.Contains(d.Location, "registry.speakeasyapi.dev")
+}
```

Comment:
Is this method needed elsewhere?

Reply:
It would be used in the CLI code where we do things like IsRemoteDocument